src/core/thread/mle.hpp

```diff
@@ -1392,15 +1392,16 @@ class Mle
      */
     typedef struct NetworkInfo
     {
-        DeviceState          mDeviceState;                ///< Current Thread interface state.
-
-        uint8_t              mDeviceMode;                 ///< Device mode setting.
-        uint16_t             mRloc16;                     ///< RLOC16
-        uint32_t             mKeySequence;                ///< Key Sequence
-        uint32_t             mMleFrameCounter;            ///< MLE Frame Counter
-        uint32_t             mMacFrameCounter;            ///< MAC Frame Counter
-        uint32_t             mPreviousPartitionId;        ///< PartitionId
-        Mac::ExtAddress      mExtAddress;                 ///< Extended Address
+        DeviceState          mDeviceState;                                      ///< Current Thread interface state.
+
+        uint8_t              mDeviceMode;                                       ///< Device mode setting.
+        uint16_t             mRloc16;                                           ///< RLOC16
+        uint32_t             mKeySequence;                                      ///< Key Sequence
+        uint32_t             mMleFrameCounter;                                  ///< MLE Frame Counter
+        uint32_t             mMacFrameCounter;                                  ///< MAC Frame Counter
+        uint32_t             mPreviousPartitionId;                              ///< PartitionId
+        Mac::ExtAddress      mExtAddress;                                       ///< Extended Address
+        uint8_t              mMlIid[OT_IP6_ADDRESS_SIZE - OT_IP6_PREFIX_SIZE];  ///< IID from ML-EID
```

Comment:
@nibanks, just to be clear, do you have enough of a deployed base for backwards compatibility to matter?  I think it would help if people need to be aware that backwards compatibility needs to be considered moving forward.  I certainly wasn't aware that backwards compatibility was important to anyone yet, given that we have not made an official release yet.